organization/canva.md

```diff
@@ -0,0 +1,74 @@
+# pyOpenSci uses Canva as a shared graphic platform
```

Comment:
Is the cCanva content in this PR intentional? there is a merge conflict with it (we merged #74 ) in this PR.